aptos-move/aptos-vm/src/aptos_vm.rs

```diff
@@ -484,6 +486,10 @@ impl AptosVM {
         let session_output = session.finish().map_err(|e| e.into_vm_status())?;
         let change_set_ext = session_output.into_change_set(&mut ())?;
 
+        // Charge gas for write set
+        gas_meter.charge_write_set_gas(change_set_ext.write_set().iter())?;
+        // TODO(Gas): Charge for aggregator writes
```

Comment:
What does this mean?

Reply:
Currently we're not charging for the delta change set generated by the aggregator.

Reply:
Ok makes sense. This sounds like something we can roll into the upgrade along with charging the multiplier for total storage usage?

Reply:
Yes. IMO the aggregator is only used within the framework right now, so it doesn't feel like a P0 to me.

Reply:
I think that makes sense, only through the Aptos coin burning and minting the aggregator for total supply should be added and subtracted. Probably reading as well, but if there are/ will be any other APIs for just reading that value, it probably should be charged (maybe even expensive since it will trigger PE conflicts, but that's a whole another story anyway for incorporating parallel execution behavior into the gas scheme - hopefully a project future game theory experts :-)

Reply:
@gelash sounds good to me. We should definitely find some time together, to take a look at the aggregator and parallel executor from at a high level and discuss what architectural changes are needed to address many of the issues we encountered.

Reply:
absolutely, that would be sweet because we also want to discuss the VM error handling and other idiosyncrasies George encountered that are in the way of some of the cool generalizations (and opening it up to non-system uses). But thankfully it looks increasingly likely that we can have all the critical features in now and evolve them with upgradability.

Reply:
Oh I just realized that not charging for delta set means just adds and subs that are definitely anyway caused by system tracking coin supply for anything. Reads materialize, so you'd be charging for it. neat.